### PR TITLE
InsertWordUI.js showAlert functions makes the Add Word container expa…

### LINKED ISSUE #17: InsertWordUI.js showAlert functions makes the Add Word container expand.

### PR DESCRIPTION
**Pull request by tiffanygan**
…nd, closes #17

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -21,7 +21,8 @@ main {
 #word-main,
 #login-main,
 #register-main,
-#settings-main {
+#settings-main,
+#insert-word-main {
   margin-top: 5rem;
 }
```
Comment (ganneo) [inline assets/css/style.css]: Don't use hard coded value

**Review by ganneo — CHANGES_REQUESTED** [review]
Address comments

**Review by ganneo — APPROVED** [review]
Perfect!

**Comment by yugan2005** [comment]
LGTM. Shipit.

**Pull request merged**